Keen/KeenClient.cs

```diff
@@ -970,5 +978,33 @@ public JObject Query(string queryName, Dictionary<string, string> parms)
                 throw ex.TryUnwrap();
             }
         }
+
+        /// <summary>
+        /// </summary>
+        public void CreateAccessKey(AccessKey.AccessKey accessKey)
```

Comment:
Does this need the namespace qualification?

Reply:
Yes since namespace and class have the same name, even though I used the namespace, C# fails to understand, `AccessKey` is a class.

Reply:
If that's the case it seems like we should rename to `AccessKeys`.

Reply:
Do you mean the namespace? We have another class called `AccessKeys`

Reply:
Yeah, or does that cause other problems?

Reply:
Yeah in that sense, we may need to go for `AccessKeys = new AccessKeys.AccessKeys(_prjSettings, keenHttpClientProvider);` in the KeenClient.cs